File: hermes.of.cheese.backend/src/main/java/com/sena/proyect/hermes/of/cheese/persistence/entity/Product.java
package com.sena.proyect.hermes.of.cheese.persistence.entity;

import lombok.Getter;
import lombok.Setter;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Getter
@Setter
@Document(collection = "Product")
public class Product {
    @Id
    private String name;
    private Double quantityInInventory;
    private Double price;
}

File: HermesSenaProyect-frontend/hermes-frontend/src/app/model/product.ts
export class Product {
    name: string;
    quantityInInventory: number;
    price: number;

    constructor(name: string, quantityInInventory: number, price: number) {
        this.name = name;
        this.quantityInInventory = quantityInInventory;
        this.price = price;
    }
}
